### ISSUE TITLE
Evaluation on MOT17 dataset

### ISSUE THREAD

**Issue by navaravan** [xingyizhou/GTR#15]
Hi @xingyizhou 

Thanks for sharing the code
How did you evaluate the tracker on MOT17 dataset and got the results. Can you list the steps followed for recreating the results.
Thanks in advance.

**Comment by xingyizhou** [comment]
Please check [here](https://github.com/xingyizhou/GTR/blob/master/docs/MODEL_ZOO.md#how-to-read-the-tables).

**Comment by Huwenzjw** [comment]
Thanks for your answer. These are some steps about how to train on MOT17.  Where can I find some steps about testing on MOT17.

**Comment by akamboj2** [comment]
@Huwenzjw 

I think the instructions linked describe how to evaluate as well. Near the top of that link it says:
"To evaluate a model with a trained/ pretrained model, run
```python train_net.py --config-file /path/to/config/name.yaml --eval-only MODEL.WEIGHTS /path/to/weight.pth``` "

I downloaded the model, placed it in model/weights/ than ran:
```python train_net.py --config-file configs/GTR_MOT_FPN.yaml --eval-only MODEL.WEIGHTS models/GTR_MOT_FPN.pth```

and I'm getting the same metrics they reported.
MOTA 71.253
IDF1 75.902
HOTA 63.047 
etc...

![image](https://user-images.githubusercontent.com/33362462/178071480-6eab3711-bb6c-4e37-9adb-12703fa2c919.png)


The only caveat is the first time i ran it, I was getting an error saying it can't find datasets/mot/MOT17/seqmaps/trainval.txt, but i saw that file in gtr/tracking/seqmaps/ so i just created a symlink. I went to datasets/mot/MOT17/ directory and did ``` ln -s GTR_ROOT/gtr/tracking/seqmaps seqmaps``` and then it worked fine.

**Comment by gurkirt** [comment]
I was able to train on MOT17 including a detector. I got similar results but not exactly the same result because I could only fit a sequence length of 6 rather than 8 at training time.

**Comment by briannlongzhao** [comment]
> here

Are you evaluating on the test set (mot17_test) or half of the training set (mot17_halfval)? I was able to get the result for halfval, but for test set my test_conf0.json generated from convert_mot2coco.py (after changing SPLITS) seems not correct. It includes the sequences in the training set but not test set, e.g. MOT17-02-DPM. Is there any other step that I did not follow correctly or maybe there's something wrong with convert_mot2coco.py? Any help would be appreciated.

**Comment by wlc17662354484** [comment]
@xingyizhou 想问一下怎么样检测在测试集上的效果，我按照GTR_MOT_FPNfull.yaml以及相应的权重文件进行测试的时候，就会显示
![W9%8(BU@~74@{IZ8@ ~RXWJ](https://user-images.githubusercontent.com/126391024/230766843-1ad0a9af-73cc-4b68-909a-37f5ab074be4.png)
这个错误
